MdeModulePkg/MdeModulePkg.dsc

```diff
@@ -517,4 +517,3 @@
   MdeModulePkg/Universal/CapsulePei/CapsuleX64.inf
 
 [BuildOptions]

```

Comment:
Better to remove that change to avoid merge issues in the future.

Reply:
Same here https://github.com/acidanthera/audk/commit/9a1136d880aeee16709d9e48309c52bc2829ddd8#diff-f97bcbb084b537538f503ed16bee9c75a91adb663c18aef2bddf3910138a9d24L23